specs/macOS/spotlight-indexing.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: Starts or stops the MacOS Spotlight indexing process mds_stores.
+command: |-
+    sudo mdutil -a -i {{switch}}
+tags:
+  - Apple
+  - MacOS
+  - mds_stores
+  - Spotlight
```

Comment:
nit: Could we try to minimize the number of tags here? In general with workflows we've found fewer and more specific tags are better. Maybe Just `mds_stores` and `MacOS`?